context-free.md

```diff
@@ -2,21 +2,41 @@
 
 ##  Who is this programming language for?
 
+I don't think there's a very narrow target audience in mind. I can see the language appealing to abstract art enthusiasts even with little programming background, or to those with significant programming background who want to dabble in art. 
+
+Ultimately the language is for anyone who hears about it and is interested in making anything with it. It's also for anyone who wants to see images created by it.
```

Comment:
I think the question relates to what are the distinguishing features of this language and who would that entice into using the language.